tests/ImageSharp.Tests/Processing/Processors/Binarization/BinaryThresholdTest.cs

```diff
@@ -53,5 +54,34 @@ public void ImageShouldApplyBinaryThresholdInBox<TPixel>(TestImageProvider<TPixe
                 ImageComparer.Tolerant().VerifySimilarityIgnoreRegion(source, image, bounds);
             }
         }
+
+        [Theory]
+        [WithFileCollection(nameof(CommonTestImages), nameof(BinaryThresholdValues), PixelTypes.Rgba32)]
+        public void ImageShouldApplyBinarySaturationThresholdFilter<TPixel>(TestImageProvider<TPixel> provider, float value)
+            where TPixel : unmanaged, IPixel<TPixel>
+        {
+            using (Image<TPixel> image = provider.GetImage())
+            {
+                image.Mutate(x => x.BinaryThreshold(value, BinaryThresholdValueType.Saturation));
+                image.DebugSave(provider, value);
```

Comment:
@Franz1960 can you please double check if the test results saved to your local `tests/Images/ActualOutput/BinaryThresholdTest/` meet your expectations (Saturation vs Luminance)?

Normally we'd have a validation call here in these functional tests (`image.CompareToReferenceOutput(...)`), comparing the result to a pre-existing reference output, but unfortunately `BinaryThresholdTest` haven't been updated to have this validation.

I don't want to block this PR on this, but my testing OCD tells that we should at least visually validate the output.

Reply:
I included comparisons to ref outputs. Used a rgb image which is better fitted to saturation operations. Added a PR in 
Imagesharp.Tests.Images repo with new ReferenceOutput images. Is this the right way to bring the ref images to work?

Reply:
Yep, now use 

``` bash
git submodule foreach git pull origin master
```
In your branch root to pull those reference images in. 

I'm going to pull down your fork later and do a performance pass as I've spotted scope for some improvement.